content/2-templating/6-conditional/marko/TrafficLight.marko

```diff
@@ -0,0 +1,17 @@
+static const TRAFFIC_LIGHTS = ["red", "orange", "green"];
+<let/lightIndex = 0/>
+<let/light = TRAFFIC_LIGHTS[lightIndex]/>
```

Comment:
```suggestion
<const/light = TRAFFIC_LIGHTS[lightIndex]/>
```
derived values use `const`, otherwise the `light` does not keep in sync with the `lightIndex`